instrumentation-api-semconv/src/testBothHttpSemconv/java/io/opentelemetry/instrumentation/api/instrumenter/http/HttpClientAttributesExtractorBothSemconvTest.java

```diff
@@ -147,6 +164,10 @@ void normal() {
                 AttributeKey.stringArrayKey("http.response.header.custom_response_header"),
                 asList("654", "321")),
             entry(NetAttributes.NET_PROTOCOL_NAME, "http"),
-            entry(NetAttributes.NET_PROTOCOL_VERSION, "1.1"));
+            entry(NetAttributes.NET_PROTOCOL_VERSION, "1.1"),
+            entry(NetworkAttributes.NETWORK_TRANSPORT, "tcp"),
```

Comment:
I think this should not be captured since `network.transport` is conditionally required:

> If not default (`tcp` for `HTTP/1.1` and `HTTP/2`, `udp` for `HTTP/3`).

Reply:
Nice catch! Will fix that 👍